ext/standard/tests/strings/str_ends_with.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+str_ends_with() function - unit tests for str_ends_with()
+--FILE--
+<?php
+/* Prototype: bool str_ends_with (string $haystack, string $needle);
+   Description: Determine if $haystack ends with $needle
+*/
+$testStr = "beginningMiddleEnd";
+var_dump(str_ends_with($testStr, "End"));
+var_dump(str_ends_with($testStr, "end"));
+var_dump(str_ends_with($testStr, "en"));
+var_dump(str_ends_with($testStr, $testStr."a"));
+var_dump(str_ends_with($testStr, ""));
+var_dump(str_ends_with("", ""));
+var_dump(str_ends_with("", " "));
+var_dump(str_ends_with("\x00", ""));
+var_dump(str_ends_with("\x00", "\x00"));
+var_dump(str_ends_with("a", "\x00"));
```

Comment:
This one should rather be `("a\x00", "\x00")` and expect `bool(true)`

Reply:
This test was to make sure that the `\x00` isn't treated like an empty string. I kept this test but added the one you suggested

Reply:
There is already a test for `($testStr, "\x00")` (actually, there is only in the other file, but there should be in this file too, see my comment above), so `("a", "\x00")` doesn't add anything